shared/js/media/media_frame.js

```diff
@@ -361,13 +369,14 @@ MediaFrame.prototype._displayImage = function(backgroundImage, width, height) {
   // Query the position of the image in order to flush the changes
   // made by setPosition() above. This prevents us from accidentally
   // animating those changes when the user double taps to zoom.
-  var temp = this.image.clientLeft;
+  temp = this.image.clientLeft;
```

Comment:
Was jshint complaining about an unused variable here or something? Its annoying to have to move the declaration up to the top for this throwaway variable.  I think I'd prefer using a jshint comment here rather that moving the var temp up top, because that makes it look like some kind of important variable.